app/models/message.rb

```diff
@@ -43,9 +43,7 @@ def finished?
       (content_text.present? || content_tool_calls.present?)
   end
 
-  def not_finished?
-    !finished?
-  end
+  def not_finished? = !finished?
```

Comment:
I really like this new feature of ruby

Reply:
yeah definitely nice to have a shorthand. reminds me of Elixir